vercel.json

```diff
@@ -1,3 +1,595 @@
 {
-    "cleanUrls": true
+    "cleanUrls": true,
+    "redirects": [
+        {
+            "source": "/cow-sdk",
+            "destination": "/cow-protocol/reference/sdks/cow-sdk",
+            "statusCode": 301
+        },
+        {
+            "source": "/cow-sdk/cow-api",
+            "destination": "https://learn.cow.fi/tutorial/getting-started-order",
+            "statusCode": 301
+        },
+        {
+            "source": "/cow-sdk/getting-started-with-the-sdk",
+            "destination": "https://learn.cow.fi/tutorial/getting-started-order",
+            "statusCode": 301
+        },
+        {
+            "source": "/cow-sdk/order-meta-data-appdata",
+            "destination": "https://learn.cow.fi/tutorial/simple-app-data",
+            "statusCode": 301
+        },
+        {
+            "source": "/cow-sdk/querying-the-cow-subgraph",
+            "destination": "https://learn.cow.fi/tutorial/getting-started-order",
+            "statusCode": 301
```

Comment:
This should be 302 as discussed here https://cowservices.slack.com/archives/C04N05LJK54/p1704884465781959?thread_ts=1704827135.936669&cid=C04N05LJK54